components/director/internal/domain/api/resolver.go

```diff
@@ -81,24 +82,30 @@ func (r *Resolver) AddAPIDefinitionToPackage(ctx context.Context, packageID stri
 	}
 	defer r.transact.RollbackUnlessCommitted(tx)
 
+	log.Infof("Adding APIDefinition to package %s", packageID)
+
 	ctx = persistence.SaveToContext(ctx, tx)
 
 	convertedIn, err := r.converter.InputFromGraphQL(&in)
 	if err != nil {
+		log.Error("Error occurred while converting GraphQL input to APIDefinition. ", err.Error())
```

Comment:
in director there is https://github.com/kyma-incubator/compass/blob/69ec36ed414a25c7f3a706948efcad19eeb9031b/components/director/cmd/director/main.go#L144 error presenter that i think already logs some errors so i suspect you might be double logging all the errors right now - could you double check on that ?

Reply:
You are right. The `presenter` logs the error so there is no point logging it again in my logs. I double checked it in all places where I have added logs. 

One minor thing: the `presenter` logs the errors on `info` level. Do you think it is okay that way or should we change it to `error` level?